packages/reporter/src/runnables/runnables-store.ts

```diff
@@ -50,6 +50,7 @@ export class RunnablesStore {
    * content: RunnableArray
    */
   @observable runnablesHistory: Record<string, RunnableArray> = {}
+  @observable totalRunnables: number = 0
```

Comment:
I'm not sure this needs to be observable. I'd expect this to be static.

Reply:
I was under the assumption that it needs to be `observable` so it triggers a rerender on change. Though it is static once the run is parsed, the store is updated later